iot_config/migrations/20241203190903_solana_net_ids.sql

```diff
@@ -0,0 +1,9 @@
+-- Migration here solely for testing purposes
+-- An instance of account-postgres-sink
+-- Will alter this table depending on an on-chain struct
+CREATE TABLE IF NOT EXISTS solana_net_ids (
+    address TEXT PRIMARY KEY,
+    id INTEGER NOT NULL,
+    authority TEXT NOT NULL,
+    current_addr_offset NUMERIC NOT NULL
```

Comment:
What is this offset?

Reply:
Part of the backfill will read all existing devaddrs and set this onchain state to the largest `end_addr` of the existing net_id. Then any additional devaddrs just use this `current_addr_offset` as their `start_addr` and increment it accordingly.